hw3/02-charts.js

```diff
@@ -24,22 +64,89 @@ const borderColors = [
   'rgba(40, 159, 64, 1)',
   'rgba(210, 199, 199, 1)',
   'rgba(78, 52, 199, 1)',
+  ...moreColors,
 ];
 
+function capitalizeHouseNames(str) {
+  return str
```

Comment:
I really like the use of map here. Very elegant.

Reply:
Thank you! I thought so too, its very similar to code I've seen at work when it comes to frontend stuff other dev's have done